docs/src/pages/guides/notifications.md

```diff
@@ -9,14 +9,15 @@ Monika will send notifications to you whenever [alerts](https://hyperjumptech.gi
 
 At this moment, Monika support these channel of notifications (You can use just one or more):
 
-1. SMTP
-2. Webhook
-3. Mailgun
-4. Sendgrid
-5. Slack
-6. Whatsapp
-7. Microsoft Teams
-8. Discord
+1. [SMTP](https://hyperjumptech.github.io/monika/guides/notifications#smtp)
+2. [Mailgun](https://hyperjumptech.github.io/monika/guides/notifications#mailgun)
+3. [SendGrid](https://hyperjumptech.github.io/monika/guides/notifications#mailgun)
```

Comment:
I missed this. Link should be https://hyperjumptech.github.io/monika/guides/notifications#sendgrid.